mkdocs.yaml

```diff
@@ -44,6 +44,8 @@ nav:
   - Authenticate: how-to/auth.md
 - Data hub: 
   - data-hub/index.md
+  - Deep Origin DataFrames:
+    - Tutorial: data-hub/dataframes.md
   - How to:
```

Comment:
i intend this to be the primary way users interact with databases, so made it front and center